backend/session/store.go

```diff
@@ -64,3 +65,18 @@ func (store *Store) readRaw(r *http.Request) *sessions.Session {
 	sess, _ := store.wrappedStore.Get(r, "auth") // ignoring, because errors only fire if a bad cookie name is provided
 	return sess
 }
+
+func userIDFromSession(sess *sessions.Session) *int64 {
+	sessionData, ok := sess.Values[sessionDataKey].(*Session)
+	if ok {
+		val := sessionData.UserID
```

Comment:
This all makes sense. Is there a reason that we didn't make the UserID in sessionData nullable and just pass it through so we don't end up with the default 0 instead of nil?

Reply:
I honestly can't remember why these were set to not null.